packages/plugins/sentry/README.md

```diff
@@ -98,9 +98,35 @@ const sentryInstance = strapi
   .getInstance();
 ```
 
-## Disabling
+## Disabling for non-production environments
 
-This plugin can be disabled in the plugins configuration file. With the `env` utility, it is also possible to disable it depending on the environment.
+The plugin will only send errors to Sentry when the `dsn` config property is set. If the `dsn` property is set to a nil value, the Sentry plugin will still be available to use in the running Strapi instance, but the service it will not actually send errors to Sentry.  
+When you start Strapi with a nil `dsn` config property, the plugin will print a warning:  
+`info: @strapi/plugin-sentry is disabled because no Sentry DSN was provided`
```

Comment:
Hi @innerdvations,
thanks a lot for your suggestion. Yes, somehow my original formulation was a little weak on emphasizing the reasoning behind this. I have now applied your suggestion apart from that nil/null change, if that's ok. Because both `null` and `undefined` result in the described behavior.

Reply:
I was switching to null since that's what was used in the example and since nil isn't clearly defined for javascript (and in fact I didn't see it used anywhere else in the documentation).  Do you know if {} or [] will work the same as null and undefined? If so, I'm ok with nil, otherwise we may want to go with falsy.

Reply:
Maybe leaving it at `null` would have been more consistent and in line with the rest of the Strapi doc, I can see that now. Don't know how strict you guys are regarding such things. Next time, I'll definitely consider such things.

Regarding falsy:
Actually, the [Strapi Sentry service](https://github.com/strapi/strapi/blob/main/packages/plugins/sentry/server/services/sentry/index.js#L25) does indeed only check for truthy/falsy. So setting the dsn to e.g. `{}` or `[]` it would indeed continue to configure Sentry with that DSN. However Sentry itself throws an exception in this case, which is caught by the Strapi Sentry service and Sentry would not log errors in this case.